### PR TITLE
OLS-1070: Update About Azure OpenAI configuration with Lightspeed

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Affects:
[lightspeed-main](https://github.com/openshift/openshift-docs/tree/lightspeed-docs-main)
[lightspeed-docs-1.0tp1](https://github.com/openshift/openshift-docs/tree/lightspeed-docs-1.0tp1)

This PR is part of the standalone doc set for the Lightspeed project. Kathryn is aware that this content applies for a product that is part of a Developer Preview release. The project is seeking feedback from early adopters.

Version(s): Tech Preview

Issue: https://issues.redhat.com/browse/OLS-1070

Link to docs preview:
https://82064--ocpdocs-pr.netlify.app/openshift-lightspeed/latest/install/ols-installing-openshift-lightspeed#about-azure-openai-configuration-with-openshift-lightspeed

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/ols-large-language-model-configuration-overview.adoc [hunk 1]
```diff
@@ -15,7 +15,7 @@ For more information, see the official {openai} link:https://platform.openai.com
 
 == About Azure OpenAI configuration with OpenShift Lightspeed
 
-You must have an existing deployment of {azure-openai}, and you must know which subscription the {azure-openai} deployment is associated with.
+You need a {azure-openai} service instance. There must be at least one model deployment in {azure-studio} for that instance.
```
Comment (thoraxe) [inline modules/ols-large-language-model-configuration-overview.adoc:18]: You need *an* (I think)

**Review by thoraxe — CHANGES_REQUESTED** [review]

File: modules/ols-large-language-model-configuration-overview.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ols-large-language-model-configuration-overview.adoc:18]: If it were Azure OpenAI, yes. It actually renders as Microsoft Azure OpenAI, so it should be a, but I see why you thought that looking at the code.

**Comment by JoaoFula** [comment]
LGTM

**Comment by stevsmit** [comment]
Lgtm

**Pull request merged**